cli.go

```diff
@@ -401,20 +401,20 @@ Options:
   -ssl-verify                  Verify certificates when connecting via SSL
   -token=<token>               Sets the Consul API token
 
-  -log-level=<level>           Set the logging level - valid values are "debug",
+  -log-level=<level>           Sets the logging level - valid values are "debug",
                                "info", "warn" (default), and "err"
   -syslog                      Send the output to syslog instead of standard error
                                and standard out. The syslog facility defaults to
                                LOCAL0 and can be changed using a configuration file
-  -syslog-facility=<f>         Set the facility where syslog should log. If this
+  -syslog-facility=<f>         Sets the facility where syslog should log. If this
                                attribute is supplied, the -syslog flag must also be
                                supplied.
 
-  -wait=<duration[:duration]>  Sets the 'minumum[:maximum]' amount of time to wait
+  -wait=<duration[:duration]>  Sets the 'minimum[:maximum]' amount of time to wait
                                before writing a triggering a restart
```

Comment:
There is still an error here: "before writing a triggering a restart". What is the intended wording?